pandas/tests/indexes/datetimes/test_date_range.py

```diff
@@ -548,85 +548,116 @@ def test_range_closed(self, freq):
         begin = datetime(2011, 1, 1)
         end = datetime(2014, 1, 1)
 
-        closed = date_range(begin, end, closed=None, freq=freq)
-        left = date_range(begin, end, closed="left", freq=freq)
-        right = date_range(begin, end, closed="right", freq=freq)
+        both = date_range(begin, end, inclusive="both", freq=freq)
```

Comment:
can you parameterize this test instead, using the `inclusive_endpoints_fixture` fixtures (its in pandas/tests/frame` so likely need to move to `pandas/conftest.py`

Reply:
yea makes sense, will do that